LICENSE

```diff
@@ -0,0 +1,15 @@
+DevOpsBootcamp
+Copyright (C) 2018
```

Comment:
This should be from ``2014 - 2017`` and you should also add ``Oregon State University`` as the author at the end.